spec/comparators/hashed_configuration_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../spec_helper'
+
+describe HashedConfiguration do
```

Comment:
HashedConfiguration is not a powerful, core abstraction and it is in fact just an approach to remove one line repeated code. By testing any comparator, you are actually testing this code.